app/Http/Controllers/CompanyController.php

```diff
@@ -34,7 +33,7 @@ public function draw(Request $request){
         }
 
         if(!empty($arr)){
-            $searchCountry = empty($arr) ? $search : (array_key_exists(3, $arr) ? $arr[3] : null);
+            $searchCountry = array_key_exists(3, $arr) ? $arr[3] : null;
```

Comment:
use the index from parameter which sends from datatable. batter not to hardcode here. i am not able to guess key 3 represents what

Reply:
Yeah okay i refactored it using column names now the code is more readable please review again @Raza9798  #79